maps/away/ships/konyang/konyang_wreck/konyang_wreck.dm

```diff
@@ -0,0 +1,159 @@
+/datum/map_template/ruin/away_site/konyang_wreck
+	name = "Konyang Wrecked Cargo Ship"
+	id = "konyang_wreck"
+	description = "Orion Express cargo ship, home to an unfortunate outbreak of infected IPCs."
+	suffixes = list("ships/konyang_wreck/konyang_wreck.dmm")
+	spawn_cost = 1
+	spawn_weight = 1
+	sectors = list(SECTOR_HANEUNIM)
+
+	unit_test_groups = list(3)
+
+/obj/effect/overmap/visitable/ship/konyang_wreck
+	name = "Konyang Wreck"
+	desc = "An Orion Express Packhorse-class freighter. This one appears to be drifting. Sensors register no life signs aboard."
+	class = "OEV"
+	icon_state = "freighter_large"
+	moving_state = "freighter_large_moving"
+	colors = list("#c3c7eb", "#a0a8ec")
+	designer = "Orion Express"
+	volume = "41 meters length, 36 meters beam/width, 11 meters vertical height"
+	sizeclass = "Packhorse-class cargo freighter"
+	shiptype = "Long-range cargo transport"
+	vessel_mass = 5000
+	max_speed = 1/(2 SECONDS)
+	burn_delay = 1 SECONDS
+	vessel_size = SHIP_SIZE_SMALL
+	initial_generic_waypoints = list(
+		"konyang_wreck_nav1",
+		"konyang_wreck_nav2",
+		"konyang_wreck_nav3",
+		"konyang_wreck_nav4",
+		"konyang_wreck_dock1",
+		"konyang_wreck_dock2",
+		"konyang_wreck_dock3",
+		"konyang_wreck_dock4"
+	)
+	fore_dir = SOUTH
+
+/obj/effect/overmap/visitable/ship/konyang_wreck/New()
+	designation = "[pick("Hauler", "Special Delivery", "50% Off Shipping", "Courier", "Telegram")]"
+	..()
```

Comment:
First call parent, then set whatever, otherwise any parent could overwrite the name you set here